macros/dune/source.sql

```diff
@@ -1,9 +1,12 @@
-{% macro source(source_name, table_name) %}
+{% macro source(source_name, table_name, dune=False) %}
```

Comment:
Could we change this to be `database="delta_prod"` and then we can override it with `database="dune"` where needed.  that way the code below will only change to `database=database` and we don't need the extra if.

Reply:
done, good call

we may eventually be able to remove the `if trino` concept altogether, but i'll save for later. i have a task in triage to go through all references to catalog usage in macros to document/understand/enhance